### PR TITLE
Add Fraxtal full config

### PR DESCRIPTION
**Pull request by vincfurc**
Resolves L2B-4061

### CODE REVIEW

File: packages/config/src/layer2s/fraxtal.ts [hunk 1]
```diff
@@ -1,17 +1,62 @@
...
+const upgradeability = {
+  upgradableBy: ['ProxyAdmin'],
+  upgradeDelay: 'No delay',
+}
+
+export const fraxtal: Layer2 = opStack({
+  daProvider: {
+    name: 'External',
+    riskView: {
+      value: 'External - Fraxtal Custom Module',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/fraxtal.ts:18]:
```suggestion
      value: 'External',
```

Comment (lucadonnoh) [reply packages/config/src/layer2s/fraxtal.ts:18]: are there any DA attestations published? if yes it should be specified, if no same thing, see Celestia for example
image

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/fraxtal.ts [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
+      description:
+        'Fraxtal uses a separate data availability module developed by the Frax Core Team. The data needed for proof construction is not published on-chain.',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/fraxtal.ts:20]:
```suggestion
        'Fraxtal uses a separate data availability module developed by the Frax Core Team. The data needed for proof construction is not published onchain.',
```

just a style nitpick

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/fraxtal.ts (hunk 2 above)
Comment (vincfurc) [reply packages/config/src/layer2s/fraxtal.ts]: Thanks, updated

File: packages/config/src/layer2s/fraxtal.ts (hunk 1 above)
Comment (vincfurc) [reply packages/config/src/layer2s/fraxtal.ts]: Thanks, added

**Review by luke7211 — APPROVED** [review]

File: packages/config/src/layer2s/fraxtal.ts [hunk 3]
```diff
@@ -24,12 +69,61 @@ export const fraxtal: Layer2 = underReview({
...
+    }),
+    discovery.getContractDetails('frxETHMinter', {
+      description:
+        'Authorized minter contract for frxETH, accepts user-supplied ETH and converts it to frxETH.',
+    }),
+    discovery.getContractDetails('sfrxETH', {
+      description:
+        'Vault token contract (ERC-4626) for staked frxETH. The smart contract receives frxETH tokens and mints sfrxETH tokens.',
+    }),
+    discovery.getContractDetails('Timelock', {
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/fraxtal.ts:123]: specify the delay

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/fraxtal.ts (hunk 3 above)
Comment (vincfurc) [reply packages/config/src/layer2s/fraxtal.ts:130]: added, thanks

File: packages/config/src/layer2s/fraxtal.ts [hunk 4]
```diff
@@ -1,17 +1,69 @@
...
-export const fraxtal: Layer2 = underReview({
-  id: 'fraxtal',
+const discovery = new ProjectDiscovery('fraxtal')
+
+const timelockDelay = discovery.getContractValue<number>(
+  'Timelock',
+  'delay',
+)
+
+const timelockDelayHours = timelockDelay / 3600
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/fraxtal.ts:14]: use `formatSeconds`

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**